ehitchhiking-ui/src/app/shared/components/chat/chat.component.ts

```diff
@@ -0,0 +1,58 @@
+import {Component, OnInit} from '@angular/core';
+import {$} from 'protractor';
```

Comment:
you don't use $, pl remove import statement

Reply:
removed